src/sentry/models/groupowner.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import absolute_import
+
+from enum import Enum
+
+from django.conf import settings
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import FlexibleForeignKey, Model
+
+
+class GroupOwnerType(Enum):
+    SUSPECT_COMMIT = 0
+    OWNERSHIP_RULE = 1
+
+
+class GroupOwner(Model):
+    """
+    Tracks the "owners" or "suggested assignees" of a group.
+    """
+
+    __core__ = False
+
+    group = FlexibleForeignKey("sentry.Group", unique=True, db_constraint=False)
+    project = FlexibleForeignKey("sentry.Project", null=False, db_constraint=False)
+    organization = FlexibleForeignKey("sentry.Organization", null=False, db_constraint=False)
+    owner_type = models.PositiveSmallIntegerField(null=False)
+    user = FlexibleForeignKey(settings.AUTH_USER_MODEL, related_name="sentry_owner_set", null=True)
```

Comment:
Do you actually need the related names? Or was this just from copy/paste?

Reply:
I thought they _could_ come in handy but I don't currently see a use case. Left them in "just in case""

Reply:
(Removed as per Slack)